rtl/src/test/scala/tensil/Scratch.scala

```diff
@@ -10,13 +10,549 @@ import chisel3.util.Decoupled
 import chiseltest._
 import tensil.util.decoupled.Counter
 import chisel3.util.Queue
+import chisel3.util.DecoupledIO
+import tensil.util.zero
 
-class Scratch extends Module { val io = IO(new Bundle {}) }
+class Scratch(depth: Int) extends Module {
```

Comment:
What Scratch stands for?

Reply:
This class is just for prototyping and testing ideas - everything here was moved over to LockPool so these changes can be deleted.